shenyu-register-center/shenyu-register-server/shenyu-register-server-nacos/src/main/java/org/apache/shenyu/register/server/nacos/NacosServerRegisterRepository.java

```diff
@@ -165,7 +165,11 @@ private void publishMetadata(final String data) {
     }
 
     private void refreshURIService(final RpcTypeEnum rpcType, final String serviceName) {
-        for (String contextPath: uriServiceCache.get(serviceName)) {
+        ConcurrentSkipListSet<String> services = uriServiceCache.get(serviceName);
+        if (services == null) {
```

Comment:
I think 
````
if (services != null) {
 .....
}
````
would be better

Reply:
It's fine for me. can avoid adding a layer of `if` nesting

Reply:
nop, I mean use if (services != null)  instead of 
````
if (services == null) {
    return;
}
````
The code will be cleaner.

Reply:
Agree, or you can use `optional`.

Reply:
maybe in this way?
`
ConcurrentSkipListSet<String> services = uriServiceCache.get(serviceName);
        Optional.ofNullable(services).ifPresent(data -> {
            for (String contextPath : data) {
                registerURI(contextPath, serviceName, rpcType);
            }
});
`

Reply:
maybe in this way?

```java
ConcurrentSkipListSet<String> services = uriServiceCache.get(serviceName);
Optional.ofNullable(services).ifPresent(data -> {
    for (String contextPath : data) {
        registerURI(contextPath, serviceName, rpcType);
    }
});
```

Reply:
maybe like this :

```java

        Optional.ofNullable(uriServiceCache.get(serviceName)).ifPresent(services -> services.forEach(contextPath-> registerURI(contextPath, serviceName, rpcType)));

```

Reply:
Can you fix the other IDEA warnings for this class?

Reply:
It's all settled